adding preview/compile updates (#3942)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md

The console section, located below the File editor, includes various console tabs and buttons to help you with tasks such as previewing, compiling, building, and viewing the <Term id="dag" />. Refer to the following sub-bullets for more details on the console tabs and buttons.
<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-console-overview.jpg" width="100%" title="The Console section is located below the File editor and has various tabs and buttons to help execute tasks"/>

> 1. **Preview button &mdash;** When you click on the Preview button, it runs the SQL in the active file editor regardless of whether you have saved it or not and sends the results to the **Results** console tab. From dbt v1.6 or higher, you can preview a selected portion of saved or unsaved code by highlighting it and then clicking the **Preview** button.

Reviewer: you can highlight and run just that code with any dbt version.

> 1. **Preview button &mdash;** When you click on the Preview button, it runs the SQL in the active file editor regardless of whether you have saved it or not and sends the results to the **Results** console tab. From dbt v1.6 or higher, you can preview a selected portion of saved or unsaved code by highlighting it and then clicking the **Preview** button.

Author: ```suggestion
1. **Preview button &mdash;** When you click on the Preview button, it runs the SQL in the active file editor regardless of whether you have saved it or not and sends the results to the **Results** console tab. You can preview a selected portion of saved or unsaved code by highlighting it and then clicking the **Preview** button.
```

> 1. **Preview button &mdash;** When you click on the Preview button, it runs the SQL in the active file editor regardless of whether you have saved it or not and sends the results to the **Results** console tab. Starting from dbt v1.6 or higher, you can preview a selected portion of saved or unsaved code by highlighting it before clicking the **Preview** button.

Author: ```suggestion
1. **Preview button &mdash;** When you click on the Preview button, it runs the SQL in the active file editor regardless of whether you have saved it or not and sends the results to the **Results** console tab. From dbt v1.6 or higher, you can preview a selected portion of saved or unsaved code by highlighting it and then clicking the **Preview** button.
```

</ul>
</details>

> 2. **Compile button &mdash;** The **Compile** button compiles the saved or unsaved SQL code and displays it in the **Compiled Code** tab. From dbt version 1.6 onwards, when you save changes to a model, you can compile its code with the model's specific context. This context is similar to what you'd have when building the model and involves useful context variables like `{{ this }} `or `{{ is_incremental() }}`.

Author: ```suggestion
2. **Compile button &mdash;** The **Compile** button compiles the saved or unsaved SQL code and displays it in the **Compiled Code** tab.
```

> Starting from dbt v1.6 or higher, after saving changes to a model, you can compile its code with the context of that model — same as you would get while building it — which includes context variables like `{{ this }}` or `{{ is_incremental() }}`.

Reviewer: yes!

> Starting from dbt v1.6 or higher, after saving changes to a model, you can compile its code with the context of that model — same as you would get while building it — which includes context variables like `{{ this }}` or `{{ is_incremental() }}`.

Author: ```suggestion
Starting from dbt v1.6 or higher, when you save changes to a model, you can compile its code with the model's specific context. This context is similar to what you'd have when building the model and involves useful context variables like `{{ this }} `or `{{ is_incremental() }}`.
```

</ul>
</details>

> 2. **Compile button &mdash;** The **Compile** button compiles the saved or unsaved SQL code from the active file editor, and outputs it to the **Compiled Code** tab.  Starting from dbt v1.6 or higher, you can compile a selected portion of saved or unsaved code by highlighting it before clicking the **Compile** button.

Reviewer 2: As above - this isn't quite right. This suggestion isn't perfect wording, but gets closer to what the substantive change is.

```suggestion
2. **Compile button &mdash;** The **Compile** button compiles the saved or unsaved SQL code from the active file editor, and outputs it to the **Compiled Code** tab. Starting from dbt v1.6 or higher, after saving changes to a model, you can compile its code with the context of that model — same as you would get while building it — which includes context variables like `{{ this }}` or `{{ is_incremental() }}`.
```

> 2. **Compile button &mdash;** The **Compile** button compiles the saved or unsaved SQL code from the active file editor, and outputs it to the **Compiled Code** tab. Starting from dbt v1.6 or higher, after saving changes to a model, you can compile its code with the context of that model — same as you would get while building it — which includes context variables like `{{ this }}` or `{{ is_incremental() }}`.

Author: ```suggestion
2. **Compile button &mdash;** The **Compile** button compiles the saved or unsaved SQL code and displays it in the **Compiled Code** tab. From dbt version 1.6 onwards, when you save changes to a model, you can compile its code with the model's specific context. This context is similar to what you'd have when building the model and involves useful context variables like `{{ this }} `or `{{ is_incremental() }}`.

Starting from dbt v1.6 or higher, after saving changes to a model, you can compile its code with the context of that model — same as you would get while building it — which includes context variables like `{{ this }}` or `{{ is_incremental() }}`.
```
